domain/crossmodelrelation/service/remoteapplication.go

```diff
@@ -139,9 +139,9 @@ type ModelRemoteApplicationState interface {
 	//     is not a remote offerer application.
 	GetOffererModelUUID(ctx context.Context, appName string) (coremodel.UUID, error)
 
-	// IsApplicationConsumer checks if the given application exists in the model and
-	// is a non-synthetic application, in the consumer model.
-	IsApplicationConsumer(ctx context.Context, appName string) (bool, error)
+	// IsApplicationLocal checks if the given application exists in the model and
+	// is a non-synthetic application.
+	IsApplicationLocal(ctx context.Context, appName string) (bool, error)
```

Comment:
todo: rename. Suggestion `IsApplicationSynthetic`, then check it returns false.

It'd be better to keep with the `synthetic` word usage than going back to the local/remote wording, we've been trying to use.